src/reducers/index.js

```diff
@@ -1,21 +1,23 @@
-import { INCREMENT, DECREMENT } from '../actions';
-
-const initialState = {
-  count: 0
-}
+import { INCREMENT, DECREMENT } from "../actions";
 
 // Our reducer that handles our two action cases:
 // increment and decrement. It receives the state
+
 // of our redux store, along with an action created
 // by our action creator. What does the reducer
+
 // need to do with the count in each case?
-export default (state = initialState, action) => {
+
+export default (count = 0, action) => {
```

Comment:
I would preserve the initial structure of this data. Right now this works because you have a single value, but what if the store consists of numerous values?